js/components/portal-dashboard/header-menu.tsx

```diff
@@ -60,39 +67,24 @@ export class HeaderMenuContainer extends React.PureComponent<IProps, IState> {
   render() {
     return (
       <div className={css.headerMenu} data-cy="header-menu" onClick={this.handleMenuClick}>
-        {this.renderMenuIcon()}
+        { this.state.showMenuItems
+          ? <CloseIcon className={`${css.icon} ${css.menuIcon}`} />
+          : <MenuIcon className={`${css.icon} ${css.menuIcon}`} />
+        }
         {this.renderMenuItems()}
         {/* <div className={css.hamBurger}></div> */}
```

Comment:
Somthing we don't need?

Reply:
I removed it.  It must have been a test or something that sneaked past us?